### ISSUE TITLE
Now it's time to implement Int8 (known from the RTX 2080 Ti) for Leela

### ISSUE THREAD

**Issue by all12** [LeelaChessZero/lc0#447]
This will double Leelas speed from 30k total nodes to 60k total nodes and also improve Leelas strength by +50 elo.
Should be done before v0.19.0 is out.

**Comment by all12** [comment]
And can this be fixed somehow?

**Comment by oscardssmith** [comment]
Not easily. the basic problem is that int8 inherently is less precise thank float16. On gpus that support it, a 40b int8 would probably be stronger per time thana 20b float16, but it would suck for everyone who's hardware didn't support it. my guess is that the solution is to wait a year or two until more gpus have support

**Comment by all12** [comment]
It looks like this problem was fixed in the Scorpio engine.

**Comment by mooskagh** [comment]
It's not considered immediately useful, so I'm inclined to close this for now to de-clutter open issues.
It's more or less proven that quantization of existing net doesn't work.
We'll probably have to train a separate int8 net when those GPUs will be more widespread, but that's out of scope now.

**Comment by oscardssmith** [comment]
I'd rather this stay open. Int8 works for everything 10 series plus, and should be stronger as we could have 2x as many blocks per time.
